index.js

```diff
@@ -89,11 +90,29 @@ const getCompiledFileSync = (moduleDir, path) => {
   return transformSrc(moduleDir, res.toString())
 }
 
+const getBsConfigModuleOptions = buildDir => {
+  const bsconfig = readBsConfigSync(buildDir)
```

Comment:
It might be a good idea to cache the result of this function. It's not expected to change for the process of the build, and this is run once per file for every build

Reply:
read-bsconfig will cache a bsconfig for us now so it doesn't read more than once